README.md

```diff
@@ -27,14 +27,14 @@ Building REEF
 Requirements
 ------------
 
-  * Java 7 Development Kit
+  * Java 7 Development Kit or newer.
```

Comment:
I think REEF usually runs on JDK 7 since HDI/HDP(2.1 and 2.2) officially supports JDK 7 only. 
In fact, I tested REEF tests compiled by JDK8 on our docker images using JDK 7.
So, for this issue, I will remove 'or newer' and do some investigation first.

I think I need to create a JDK8-based docker image based on HDP 2.3 which officially supports JDK 7 and 8.